paddle/scripts/docker/Dockerfile

```diff
@@ -12,6 +12,14 @@ RUN apt-get update \
 RUN pip install -U BeautifulSoup docopt PyYAML pillow \
     sphinx sphinx_rtd_theme breathe recommonmark
 
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
I am curious why ARG in Dockerfile can surpress errors from programs running in there? Did you learn this trick from somewhere? Can you please share a link or something?

I tried your change without the ARG.  Actually it works! :-)